README.md

```diff
@@ -72,54 +76,72 @@ We host weekly community meetings at 1pm EST on Wednesdays. These community meet
 
 We host our own [community slack](https://slack.wasmcloud.com) for all community members to join and talk about WebAssembly, wasmCloud, or just general cloud native technology. For those of you who are already on the [CNCF Slack](https://cloud-native.slack.com/), we also have our own channel at [#wasmcloud](https://cloud-native.slack.com/archives/C027YTXEYFL).
 
-## Reference Documentation
+---
+
+## 📚 Reference Documentation
 
 wasmCloud uses some terminology you might not be familiar with. Check out the [reference](https://wasmcloud.com/docs/category/reference) section of our docs for a deeper dive.
 
+---
+
 ## RPC Framework
 
-wasmCloud uses [wasmbus-rpc](https://github.com/wasmCloud/weld/tree/main/rpc-rs) to communicate between the host runtime, actors, and providers.
+wasmCloud uses an [RPC API](https://wasmcloud.com/docs/hosts/lattice-protocols/rpc) to enable seamless communication among the host runtime, actors, and providers.
+
+---
 
 ## Declarative Deployments
 
 The **w**asmCloud **A**pplication **D**eployment **M**anager [wadm](https://github.com/wasmCloud/wadm) uses the Open Application Model to define and deploy application specifications.
 
-## Host Runtimes
-
-### ☁️ Elixir/OTP Runtime
+---
 
-The primary Cloud Native wasmCloud host runtime is the [Elixir/OTP](https://github.com/wasmCloud/wasmcloud-otp) runtime. wasmCloud leverages Elixir/OTP for its battle-tested, massively-scalable foundation; we leverage Rust for its zero-cost abstractions, safety, security, and WebAssembly support.
+## Host Runtimes
 
-### 🦀 Rust Runtime (`Experimental`)
+### 🦀 Rust Runtime
 
-Rust runtime is under heavy development at the root of this repository.
+wasmCloud's [standard runtime](./crates/runtime) is built in Rust for its zero-cost abstractions, safety, security, and WebAssembly support.
 
 ### 🕸 JavaScript Runtime (`Experimental`)
 
 For running a wasmCloud host in a browser or embedding in a JavaScript V8 host, use the [JavaScript Runtime](https://github.com/wasmCloud/wasmcloud-js)
 
+~~### ☁️ Elixir/OTP Runtime (`Deprecated`)~~
```

Comment:
Not sure I really like the strikethrough on this section, it also causes the header to not render

Reply:
Oh good catch, didn't notice that